webapp/lib/view.dart

```diff
@@ -56,6 +56,34 @@ void clearMain() {
   snackbarView.snackbarElement.remove();
 }
 
+class UrlView {
+
+  static const List<String> _queryFilterKeys = ['type', 'project'];
```

Comment:
I don't think you need to list here the query keys, as you're still having to check if they've been set or not in `changeViewOnUrlChange`. So I'd keep the `UrlView` blind to what parameters are being set or not, and just use it as a friendlier general interface to the URL params.

Reply:
That's being used internally in getPageUrlFilters() but I can refactor it to be blind as you say